Makefile

```diff
@@ -1,5 +1,3 @@
-QC_SOC_TARGET?="APQ8074"
-
 all: linux qurt rpi bebop edison
```

Comment:
No, that's not a merge conflict, it's on purpose.
It is now in Toolchain-qurt.cmake